makefile.js

```diff
@@ -0,0 +1,102 @@
+import {argv} from "node:process"
+import {existsSync} from "node:fs"
+import {fileURLToPath} from "node:url"
+import {join} from "node:path"
+import {mkdir, writeFile} from "node:fs/promises"
+import axios from "axios"
+import sade from "sade"
+
+/**
+ * @typedef {Object} OpenAPI
+ * @property {Paths} paths
+ * 
+ * @typedef {Object.<string, Path>} Paths
+ * 
+ * @typedef {Object.<string, Method>} Path
+ * 
+ * @typedef {{description: string, summary: string, "x-shortName": string}} Method
+ */
```

Comment:
When defining multiple types, prefer to separate each into its own comment block. Here, you do not have to create your types; just use the already existing ones from the [openapi-types](https://www.npmjs.com/package/openapi-types/) package.